The plugin is already preloaded by our test framework so we do not need to load
it again.

--- pytest_lazy_fixtures-1.4.1/tests/conftest.py.orig
+++ pytest_lazy_fixtures-1.4.1/tests/conftest.py
@@ -3,7 +3,7 @@
 from pytest_lazy_fixtures import lf
 from tests.entity import Entity
 
-pytest_plugins = ("pytest_lazy_fixtures.plugin", "pytester")
+pytest_plugins = ("pytester")
 
 
 @pytest.fixture
